glass-easel/src/element.ts

```diff
@@ -1862,7 +1873,12 @@ export class Element implements NodeCast {
       }
       ;(frag as backend.Element).release()
     } else {
-      Element.insertChildComposed(parent, replacer, placeholder, true, posIndex)
+      if (placeholder.isVirtual()) {
+        // virtual placeholder does not need to remove
+        Element.insertChildComposed(parent, replacer, undefined, false, posIndex)
```

Comment:
Looks weird. What about its composed children?

Reply:
> Looks weird. What about its composed children?

1. For domlike backend, the following `insertChildComposed` inside for block for replacedChildren will insert placeholder's composed children into the correct place without removal.
2. For composed backend, placeholder's composed children already have been removed here https://github.com/wechat-miniprogram/glass-easel/pull/90/files#diff-bb609e82d5d74cf13c6f70177433019b0f39a474147273fdb0ad234bc09264dfL1817
3. Without this change, its composed children will be removed multiple times, first at L1817, then L1865 will try for the second time.

Reply:
Understand. It is correct.